### PR TITLE
fix(neovide-bin): add app icons and entry

### PR DESCRIPTION
**Pull request by nezred**

### CODE REVIEW

File: packages/neovide-bin/neovide-bin.pacscript [hunk 1]
```diff
@@ -11,13 +11,29 @@ maintainer="wizard-28 <wiz28@pm.me>"
...
+pkgrel="2"
 pkgdesc="No nonsense neovim client in Rust"
 url="https://github.com/neovide/neovide/releases/download/${pkgver}/neovide-linux-x86_64.tar.gz"
 breaks=("${gives}" "${gives}-git" "${gives}-deb" "${gives}-app")
 hash="f2e13a9d1941ea8418b5fde5a884892202657aff0b8fcddcf540b7d4caa69c24"
 repology=("project: neovide")
 
+prepare() {
+  for s in 16 32 48 256
+  do
```
Comment (Elsie19) [inline packages/neovide-bin/neovide-bin.pacscript:23]: Wrong formatting. Run it through shfmt like we require.

File: packages/neovide-bin/neovide-bin.pacscript [hunk 2]
```diff
@@ -11,13 +11,29 @@ maintainer="wizard-28 <wiz28@pm.me>"
...
+    wget -q "https://raw.githubusercontent.com/neovide/neovide/${pkgver}/assets/${gives}-${s}x${s}.png"
+  done
+  wget -q "https://raw.githubusercontent.com/neovide/neovide/${pkgver}/assets/${gives}.desktop"
+  wget -q "https://raw.githubusercontent.com/neovide/neovide/${pkgver}/LICENSE"
+}
+
 package() {
   sudo install -vDm755 "${gives}" -t "${pkgdir}/usr/bin"
+  for s in 16 32 48 256
+  do
```
Comment (Elsie19) [inline packages/neovide-bin/neovide-bin.pacscript:33]: Same.

**Review by Elsie19 — CHANGES_REQUESTED** [review]

**Review by Elsie19 — CHANGES_REQUESTED** [review]
Tabbing is still wrong. It should be 2 spaces.

**Review by nezred — COMMENTED** [review]
okay you are right, when running shfmt through vim with `:%!shfmt`, vim used tabs and not spaces for some reason.

**Comment by Elsie19** [comment]
Pull from master.

**Review by Elsie19 — APPROVED** [review]
Thanks for the PR!

**Pull request merged**